exercises/formatSeconds/formatSeconds.js

```diff
@@ -17,10 +17,32 @@
  */
 
 function formatSeconds(num) {
-  // This is your job. :)
+  
+  // Create reference dictionary for lengths of time in seconds
+  const secondsRef = {'w':604800, 'd':86400, 'h':3600, 'm':60, 's':1};
```

Comment:
Consider defining constants with helpful names rather than [magic constants](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants)

```javascript
let SECONDS_PER_MINUTE = 60;
let SECONDS_PER_HOUR = SECONDS_PER_MINUTE * 60;
let SECONDS_PER_DAY = SECONDS_PER_HOUR * 24;
let SECONDS_PER_WEEK = SECONDS_PER_DAY * 7;
```